core_contracts/loans/loans/loans.py

```diff
@@ -724,23 +756,83 @@ def retireRedeem(self, _symbol: str, _sicx_from_lenders: int) -> None:
 
             remaining_supply -= user_debt
         price = asset.priceInLoop()
-        self.AssetRetired(self.msg.sender, _symbol, _redeemed, price, _redeemed,
+        self.AssetRetired(self.msg.sender, _symbol, _redeemed, price,
```

Comment:
We are actually lacking some information that should go in this eventlog. We should also record either the amount of sICX sold or the sICX/bnUSD price from the DEX. Probably the sICX sold. We didn't have it before since the difference from the Oracle price was given to the trader.

Also, if we are going to be changing the information returned by the eventlog let's change the name like we did for the Dividends eventlog - just add 'v2' to the name.